src/main/webapp/app/core/main/controllers/productController.js

```diff
@@ -1,8 +1,8 @@
-app.controller("ProductCtrl", function ($scope, $uibModal, $http, toastr,$location, ProductService, AuthService, ProductStatus) {
+app.controller("ProductCtrl", function ($scope, $uibModal, $http, toastr,$location, ProductService, AuthService, ProductStatus, dateFilter) {
```

Comment:
are you using this dateFilter in anywhere?

Reply:
no more, i will remove